src/components/IconTable/index.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import { css } from 'emotion';
+
+interface Icon {
+  src: string;
+  left: number;
+  top: number;
+  width: number;
+  height: number;
+  onMouseEnterHandler?: (event?: React.MouseEvent) => void;
+  onMouseExitHandler?: (event?: React.MouseEvent) => void;
+  onClickHandler?: (event?: React.MouseEvent) => void;
+}
+
+interface IconTableProps {
+  heightVW: number;
+  backgroundSrc: string;
+  icons: Icon[];
+}
+
+export default class IconTable extends React.Component<IconTableProps> {
+  constructor(props) {
```

Comment:
You don’t need a constructor if it’s just calling super

Reply:
(Kinda left there in case we want to plug in some state)